engine/apps/email/templates/email_notification.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
may be use the same format as alerting? 
@raphael-batte can you please help Vadim
probably worth to do in the separate pr so we won't block this one